specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -678,7 +684,7 @@
         },
         "type": {
           "type": "string",
-          "description": "Determines whether the IP is exposed to the public internet.",
+          "description": "The type of IP address determining whether the IP is exposed to the public internet.",
```

Comment:
Recommend simply striking "The type of IP address determining" from this, or go with something like:

"Specifies whether the IP is exposed to the public internet."

Reply:
Also, line **678** (the "ports" property description) could do with a bit more explanatory text. Currently, it reads:

"The ports."

TCP only? UDP? Both? How do ports specified here relate to those on a container instance? Doesn't have to be long and involved, but a little more on this could go a long way.